rust/src/lib/ovsdb/show.rs

```diff
@@ -80,7 +80,19 @@ fn parse_ovs_bridge_conf(
     for port_uuid in ovsdb_br.ports.as_slice() {
         if let Some(ovsdb_port) = ovsdb_ports.get(port_uuid) {
             let mut port_conf = OvsBridgePortConfig::new();
-            port_conf.name.clone_from(&ovsdb_port.name);
+            if ovsdb_port.ports.len() == 1 {
```

Comment:
Could you add a comment here explaining why this can be > 1? I know it is because of ovs-bond but it would be nice to remember it as ovs-bond is not so obvious..